### DISCUSSION TITLE
Renaming `UnArchiveAction` to `UnarchiveAction`

### DISCUSSION THREAD

**Discussion by dissto** [okeonline/filament-archivable#5] [Ideas]
It's just a thought though but I think the action should be named `UnarchiveAction` as opposed to `UnArchiveAction`. 

I'm not a native English speaker, but i'm sure that `unarchive` is one word. 🤓

**Comment by rvzug** [comment]
Yeah, fair question... I followed the naming-convention of the Archivable-package, see: https://github.com/joelbutcher/laravel-archivable/blob/1.x/src/Archivable.php#L116
